<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondPage</class>
 <widget class="QDialog" name="secondPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SecureSync GUI</string>
  </property>
  <widget class="QLineEdit" name="directoryLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseDirectoryButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose directory</string>
   </property>
  </widget>
  <widget class="QListWidget" name="fileListWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>271</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="gitInitButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>git init</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gitAddButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>git add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gitPullButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>git pull</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gitCommitButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>git commit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="commitMessageTextEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>content of the directory </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>commit message</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
